src/main/java/fi/otavanopisto/kuntaapi/server/index/AbstractIndexHander.java

```diff
@@ -40,19 +46,44 @@
   
   @Inject
   private Logger logger;
-
+  
+  @Inject
+  private GenericHttpClient httpClient;
+  
   private String index;
   private TransportClient client;
   
+  
   @PostConstruct
   public void init() {
+    debugPrint("http://localhost:9300");
+    debugPrint("http://localhost:9200");
+    
     String[] hosts = systemSettingController.getSettingValues(KuntaApiConsts.SYSTEM_SETTING_ELASTIC_SEARCH_HOSTS, DEFAULT_HOSTS);
     String clusterName = systemSettingController.getSettingValue(KuntaApiConsts.SYSTEM_SETTING_ELASTIC_CLUSTER_NAME, DEFAULT_CLUSTERNAME);
     index = systemSettingController.getSettingValue(KuntaApiConsts.SYSTEM_SETTING_ELASTIC_INDEX, DEFAULT_INDEX);
     client = createTransportClient(hosts, clusterName);
     setup();
   }
   
+  private void debugPrint(String url) {
+    try {
+      CloseableHttpClient httpClient = HttpClients.createDefault();
+      try {
+        HttpGet request = new HttpGet(url);
+        try (CloseableHttpResponse response = httpClient.execute(request)) {
+          String content = IOUtils.toString(response.getEntity().getContent());
+          System.out.println(String.format("%s replied %s", content));
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Not enough arguments. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2275)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)